examples/dreambooth/train_dreambooth_lora_sdxl.py

```diff
@@ -878,62 +874,81 @@ def load_model_hook(models, input_dir):
         eps=args.adam_epsilon,
     )
 
-    # We ALWAYS pre-compute the additional condition embeddings needed for SDXL
-    # UNet as the model is already big and it uses two text encoders.
-    # TODO: when we add support for text encoder training, will reivist.
-    tokenizers = [tokenizer_one, tokenizer_two]
-    text_encoders = [text_encoder_one, text_encoder_two]
+    # Computes additional embeddings/ids required by the SDXL UNet.
+    # regular text emebddings (when `train_text_encoder` is not True)
+    # pooled text embeddings
+    # time ids
 
-    # Here, we compute not just the text embeddings but also the additional embeddings
-    # needed for the SD XL UNet to operate.
-    def compute_embeddings(prompt, text_encoders, tokenizers):
+    def compute_time_ids():
+        # Adapted from pipeline.StableDiffusionXLPipeline._get_add_time_ids
         original_size = (args.resolution, args.resolution)
```

Comment:
```suggestion
        original_size = (args.resolution, args.resolution)
```
This should ideally be the original size of the passed image (before resizing), but ok to leave as is for now